src/main/java/RandomLocationGenerator.java

```diff
@@ -0,0 +1,11 @@
+import java.util.List;
+import java.util.Random;
+
+public class RandomLocationGenerator implements Randomizer {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All classes and interfaces must belong to a named package](https://www.codacy.com/app/mollie-c-stephenson/TicTacToe/file/2534705892/issues/source?bid=3284885&fileBranchId=3284887#l4)